pkg/api/handlers/compat/images_build.go

```diff
@@ -44,13 +44,20 @@ func genSpaceErr(err error) error {
 }
 
 func BuildImage(w http.ResponseWriter, r *http.Request) {
+	var multipart bool
 	if hdr, found := r.Header["Content-Type"]; found && len(hdr) > 0 {
 		contentType := hdr[0]
+		multipart = strings.HasPrefix(contentType, "multipart/form-data")
+		if multipart {
+			contentType = "multipart/form-data"
+		}
 		switch contentType {
 		case "application/tar":
 			logrus.Infof("tar file content type is  %s, should use \"application/x-tar\" content type", contentType)
 		case "application/x-tar":
 			break
+		case "multipart/form-data":
+			logrus.Debugf("Received multipart/form-data: %s", contentType)
```

Comment:
Isn't this always going to log "Received multipart/form-data: multipart/form-data"?